pkgs/by-name/my/mympd/package.nix

```diff
@@ -33,6 +33,7 @@ stdenv.mkDerivation (finalAttrs: {
     gzip
     perl
     jq
+    lua5_3 # luac is needed for cross builds
```

Comment:
Hmm I tried to figure out locally if we might as well remove `lua5_3` from the `buildInputs`, and I'm not sure. I noticed that it doesn't retain references to Lua, and that without `lua5_3` in the `buildInputs` both running `mympd` from the CLI works, and also the `passthru.test` works.

What do you think?

Reply:
Removing lua5_3 from buildInputs would just disable lua scripts support
```
% ldd $(nix build nixpkgs#mympd --print-out-paths)/bin/mympd | grep lua
        liblua.so.5.3 => /nix/store/1vmr66rr25djic4k8j2cx61l511f9hhx-lua-5.3.6/lib/liblua.so.5.3 (0x00007fa632ea0000)
```
https://jcorporation.github.io/myMPD/050-scripting/